### ISSUE TITLE
ExecutionException: java.lang.UnsupportedClassVersionError: org/vaadin/addon/driverjs/DriverJS has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

### ISSUE THREAD

**Issue by jcgueriaud1** [appreciated/designer-for-flow#61]
### Designer bug report Template
If you think the issue could be related to your template files please consider providing it to make it easier to reproduce your issue. 
### Stacktrace
```
java.util.concurrent.ExecutionException: java.lang.UnsupportedClassVersionError: org/vaadin/addon/driverjs/DriverJS has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at com.vaadin.flow.server.FutureAccess.get(FutureAccess.java:62)
	at com.vaadin.flow.server.VaadinService.runPendingAccessTasks(VaadinService.java:2042)
	at com.vaadin.flow.server.VaadinSession.unlock(VaadinSession.java:663)
	at com.vaadin.flow.server.VaadinService.ensureAccessQueuePurged(VaadinService.java:2005)
	at com.vaadin.flow.server.VaadinService.accessSession(VaadinService.java:1972)
	at com.vaadin.flow.server.VaadinSession.access(VaadinSession.java:951)
	at com.vaadin.flow.component.UI.access(UI.java:503)
	at com.vaadin.flow.component.UI.access(UI.java:486)
	at com.github.appreciated.designer.application.presenter.ProjectPresenter.lambda$onAttach$2(ProjectPresenter.java:108)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.UnsupportedClassVersionError: org/vaadin/addon/driverjs/DriverJS has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)...
```

Hi I've tried the application on Mac. it seems that the jvm used to run the application is java8, it's installed on my computer but the default should be java11. Does it read the java version in maven or somewhere else?

I can add a sample project if needed.

**Comment by appreciated** [comment]
A openjdk is shipped within the application. Will change the jdk from 8 to 11.

**Comment by appreciated** [comment]
closed in 7fc4853fbf3e88d889efce6d4fa2d6c8ecf1f77b
